### DISCUSSION TITLE
Font

### DISCUSSION THREAD

**Discussion by AhmedThe13thUser** [kyazdani42/nvim-web-devicons#353] [Q&A] [answered]
I have downloaded and installed a nerd font (Hack font).

but I don't know what to do.
idk if I have to use it in Neo Vim, Or if I have to use it in zsh.
if either idk what to do.

thanks.

**Comment by Akmadan23** [comment, marked as answer]
You have to set the right font in your terminal settings, so the process depends on which terminal you are using.

**Reply by AhmedThe13thUser** [reply]
Just went with the "Easier" Route of making the change in my ssh terminal.

sorry for not realizing that earlier
